### PR TITLE
Merge Dev into Assignment 5

### PR DESCRIPTION
**Pull request by JrmCkh**

### CODE REVIEW

File: frontend/src/utils/helpers.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ export const getUserInfo = async () => {
   }
 };
 
+export const getLoginStatus = async () => {
+  const response = await getUserInfo();
+  // Return true if data is not null/undefined
+  return !!response?.data?.userInfo;
```
Comment (lhy-hoyin) [inline frontend/src/utils/helpers.js:33]: Is the use of `!!` to force it to a boolean (true/false)?

**Review by lhy-hoyin — APPROVED** [review]
LGTM

File: frontend/src/utils/helpers.js (hunk 1 above)
Comment (lhy-hoyin) [reply frontend/src/utils/helpers.js:33]: Probably can do something like `return response?.data?.userInfo ? true : false;` for more clarity

Comment (JrmCkh) [reply frontend/src/utils/helpers.js:33]: To be resolved on next PR

**Pull request merged**